Gulpfile.mjs

```diff
@@ -263,9 +265,11 @@ async function buildBabel(useWorker, ignore = []) {
   for (const file of files) {
     // @example ./packages/babel-parser/src/index.js
     const dest = "./" + mapSrcToLib(file.slice(2));
-    promises.push(worker.transform(file, dest));
+    promises.push(
+      worker.transform(file, dest, { sourceMaps: enableSourceMap })
+    );
   }
-  return Promise.allSettled(promises).finally(() => {
+  return Promise.all(promises).finally(() => {
```

Comment:
This is a small fix that previously failed without prompt and continued to run.

Reply:
I change `allSettled` from `all` because all files should be compiled regardless of errors: errors in `packages/a` should not prevent `packages/b` from compiling. When using `Promise.all`, if you fix errors in `packages/a`, the watcher will not compile `packages/b` because it is not changed, but it need so.

However I forgot to iterate the return results from `Promise.allSettled` and print the rejected errors.

Reply:
I made some modifications, you try it, I'm not very familiar with `watch`.